spec/helper.js

```diff
@@ -21,6 +21,27 @@ global.onHttpContext = index.onHttpContextFactory();
 // Legacy
 global.dihelper = onHttpContext.helper;
 
+helper.httpServerBefore = function(overrides, endpointOpt) {
+    before('helper.httpServer.before', function() {
+        this.timeout(5000);
+        return helper.startServer(overrides, endpointOpt);
+    });
+
+    beforeEach(function () {
+        this.sandbox = sinon.sandbox.create();
+    });
+
+    afterEach(function () {
+        this.sandbox.restore();
+    });
+};
+
+helper.httpServerAfter = function() {
+    after('helper.httpServer.after', function() {
```

Comment:
How about moving this to `httpServerBefore`, so that the unit-test doesn't need to call the `httpServerAfter`